lib/venue.rb

```diff
@@ -1,2 +1,19 @@
 class Venue
+  attr_reader :name, :capacity
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def patrons
```

Comment:
if you add `patrons` to your `attr_reader` you can get rid of this method. Otherwise you would want to call `@patrons` in here.